Angular: add TypeScript Support topic (#5658)
DevExpress/devextreme-documentation: concepts/40 Angular Components/70 TypeScript Support/05 Component Specific Types.md

To get component-specific types, import the `DxComponentTypes` declaration where `Component` is the component name:

    <!-- tab: app.component.ts -->
    import { Component } from '@angular/core';
    import { DxDateBoxTypes } from 'devextreme-angular/ui/date-box';

    @Component({
        selector: 'app-root',
        templateUrl: './app.component.html',
        styleUrls: ['./app.component.css']
    })

    export class AppComponent {
        dateType: DxDateBoxTypes.DateType = "datetime";
    }

    <!-- tab: app.component.html -->
    <dx-date-box ...
        [type]="dateType"
    >
    </dx-date-box>

> If you need several components of the same type, you can import that type from the `common` submodule:

Author: ```suggestion
If you need the same type for multiple components, you can import that type from the `common` submodule:
```

> If you need the same type for multiple components, you can also import this type from `common` submodule:

Reviewer: ```suggestion
If you need several components of the same type, you can import that type from the `common` submodule:
```
